app/services/create_cluster_stats_service.rb

```diff
@@ -36,6 +38,27 @@ def call
 
   private
 
+  def epochs_annual_roi
+    last_epochs = last_epochs(EPOCHS_TO_CALCULATE)
+    if last_epochs.count > 0
+      epochs_duration = (last_epochs.second.created_at - last_epochs.first.created_at) / 60 / 60 / 24
+      epochs_average_roi = last_epochs(EPOCHS_TO_CALCULATE).map do |epoch|
```

Comment:
Don't we have `last_epochs(EPOCHS_TO_CALCULATE)` assigned to a variable few lines above?